app/src/main/java/org/loklak/wok/ui/suggestion/SuggestFragment.java

```diff
@@ -21,56 +22,58 @@
 
 import com.jakewharton.rxbinding2.widget.RxTextView;
 
+import org.loklak.wok.Constants;
+import org.loklak.wok.LoklakWokApplication;
+import org.loklak.wok.R;
+import org.loklak.wok.Utility;
 import org.loklak.wok.adapters.SuggestAdapter;
-import org.loklak.wok.api.LoklakApi;
-import org.loklak.wok.api.RestClient;
 import org.loklak.wok.model.suggest.Query;
-import org.loklak.wok.model.suggest.SuggestData;
 import org.loklak.wok.ui.activity.SearchActivity;
 import org.loklak.wok.ui.activity.TweetPostingActivity;
-import org.loklak.wok.R;
-import org.loklak.wok.Constants;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
+import javax.inject.Inject;
+
 import butterknife.BindString;
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 import io.reactivex.Observable;
-import io.reactivex.android.schedulers.AndroidSchedulers;
-import io.reactivex.disposables.CompositeDisposable;
-import io.reactivex.disposables.Disposable;
-import io.reactivex.schedulers.Schedulers;
-import io.realm.Realm;
-import io.realm.RealmResults;
 
 
-public class SuggestFragment extends Fragment implements SuggestAdapter.OnSuggestionClickListener {
+public class SuggestFragment extends Fragment
+        implements SuggestAdapter.OnSuggestionClickListener, SuggestContract.View{
 
-    private final String PARCELABLE_RECYCLERVIEW_STATE = "recycler_view_state";
+    private final String PARCELABLE_RECYCLER_VIEW_STATE = "recycler_view_state";
 
     @BindView(R.id.toolbar)
     Toolbar toolbar;
+
     @BindView(R.id.tweet_search_edit_text)
     EditText tweetSearchEditText;
+
     @BindView(R.id.clear_image_button)
     ImageButton clearImageButton;
+
     @BindView(R.id.refresh_suggestions)
     SwipeRefreshLayout refreshSuggestions;
+
     @BindView(R.id.tweet_search_suggestions)
     RecyclerView tweetSearchSuggestions;
+
     @BindString(R.string.network_request_error)
     String networkRequestError;
 
+    @Inject
+    SuggestPresenter suggestPresenter;
+
     private Toast mToast;
 
     private SuggestAdapter mSuggestAdapter;
 
-    private CompositeDisposable mCompositeDisposable;
-    private Realm mRealm;
-
     private final String LOG_TAG = SuggestFragment.class.getName();
 
     public SuggestFragment() {
```

Comment:
this is a default constructor, this isn't necessary

Reply:
Removed it.